src/NavBar/SelectCity/SelectCity.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import "./SelectCity.css";
+import { connect } from 'react-redux';
+import Actions from '../../Actions/actions.js';
+
+
+class SelectCity extends Component {
+    constructor(){
+        super();
+        this.state = {
+            currentSelectedCity: 'All cities',
+            isMenuShown: false,
+        }
+    }
+
+    toggleMenu(){
+        this.setState({
+            isMenuShown: !this.state.isMenuShown
+        })
+    }
+
+    handleSelectCity(city){
+        this.toggleMenu()
+        this.props.updateLocation(city)
+    }
+
+    renderSelectCityMenu(){
+        if(this.state.isMenuShown){
```

Comment:
Remember what I said about the indent. Do the check for returning null first, then the rest doesn't have to be indented